vcpkg-configuration.json

```diff
@@ -2,20 +2,16 @@
 	"registries": [
 	  {
 		"kind": "artifact",
-		"location": "https://aka.ms/vcpkg-ce-default",
-		"name": "microsoft"
-	  },
-	  {
-		"kind": "artifact",
-		"location": "https://artifacts.tools.arm.com/vcpkg-ce-registry/registry.zip",
+		"location": "https://artifacts.tools.arm.com/vcpkg-registry",
 		"name": "arm"
 	  }
 	],
 	"requires": {
-	  "arm:tools/open-cmsis-pack/cmsis-toolbox": "^2.1.1",
-	  "arm:compilers/arm/armclang": "^6.21",
-	  "arm:models/arm/avh-fvp": "^11.22.39",
-	  "microsoft:tools/kitware/cmake": "^3.25.2",
-	  "microsoft:tools/ninja-build/ninja": "^1.10.2"
+	  "arm:tools/kitware/cmake": "3.28.4",
```

Comment:
Prior to this change you have configured a compatible min version using prepending `^`. Now you are requiring a fixed version of all tools, compilers and models. Is that intentional? In that case remove `^` from arm-none-eabi-gcc as well.

Reply:
Has been solved.